.github/workflows/test-suite.yml

```diff
@@ -248,6 +248,7 @@ jobs:
         run: "composer test:adapter:xml"
 
       - name: "Test - Examples"
+      if: ${{ matrix.php-version == '8.2' && matrix.dependencies == 'locked' }}
```

Comment:
After releasing 0.11.0 this can be removed. Some examples are failing because if all dependencies are not explicitly provided like `array-dot` which a dependency of `etl` composer will not install dependencies due to `array-dot` php version constraint in release `0.10.0`

Reply:
(fixed indention)